avocado/utils/software_manager.py

```diff
@@ -848,6 +1002,55 @@ def provides(self, path):
         except process.CmdError:
             return None
 
+    def source(self, name, path):
+        """
+        Get source for provided package. Returns the path given with source.
+
+        :param name: parameter wildcard package to get the source for
+        :type name: str
+        """
+        if not self.check_installed('dpkg-dev'):
+            if not self.install('dpkg-dev'):
+                log.info("SoftwareManager (AptBackend) can't install packages "
+                         "from local .deb files with dependency resolution: "
+                         "Package 'dpkg-dev' could not be installed")
+        src_cmd = '%s source %s' % (self.base_command, name)
+        try:
+            if self.build_dep(name):
+                if not os.path.exists(path):
+                    os.makedirs(path)
+                os.chdir(path)
+                process.system_output(src_cmd)
+                for subdir in os.listdir(path):
+                    if subdir.startswith(name) and os.path.isdir(subdir):
+                        path = os.path.join(path, subdir)
+                        break
```

Comment:
`return path` can be here instead of `break` or at the end of `try:` block

Reply:
sure, will change.